icons/user-off.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "Footagesus"
```

Comment:
You should include the contributors, `tags` and `categories` from the `user` icon.

I have also pushed a fix to lucied studio, that will do this automatically do this for any future icons that we create.